client/app/core/core.module.js

```diff
@@ -23,6 +23,7 @@ import { BaseModalController } from './modal/base-modal-controller.js';
 import { BaseModalFactory } from './modal/base-modal.factory.js';
 import { ChargebackFactory } from './chargeback.service.js';
 import { CollectionsApiFactory } from './collections-api.factory.js';
+import { AutoRefreshFactory } from '../services/auto-refresh.service.js';
```

Comment:
If the factory is to be advertised on the core module it should be located in the core module folder